core/src/main/java/org/apache/accumulo/core/clientImpl/ClientContext.java

```diff
@@ -31,7 +31,6 @@
 import org.apache.accumulo.core.client.AccumuloException;
 import org.apache.accumulo.core.client.AccumuloSecurityException;
 import org.apache.accumulo.core.client.BatchWriterConfig;
-import org.apache.accumulo.core.client.ClientInfo;
```

Comment:
Unused import?

Reply:
Oh right its now in impl, so you dont need the import.

Reply:
impsort-maven-plugin FTW! :smiley_cat: